guide/fortran-compiler.rst

```diff
@@ -0,0 +1,23 @@
+##################
+Fortran90 compiler
+##################
+
+The DART software is written in standard Fortran 90, with no
+compiler-specific extensions. It has been compiled and run with several
+versions of each of the following:
```

Comment:
I believe some F95 or F2000 extensions have crept in ... the 'endian' switch on file opens, for example.